db/schema.rb

```diff
@@ -168,6 +168,16 @@
 
   add_index "pages", ["slug"], name: "index_pages_on_slug", unique: true, using: :btree
 
+  create_table "sponsorships", id: false, force: :cascade do |t|
+    t.integer  "event_id",        null: false
+    t.integer  "organization_id", null: false
```

Comment:
Can the sponsorship have a "message" field? A text type would be great. That would be VERY useful to display a customized announcement for the sponsorship.